### PR TITLE
Add lint to taskfile

### PR DESCRIPTION
**Pull request by andrlime**
title

### CODE REVIEW

File: linter/Taskfile.yml [hunk 1]
```diff
@@ -8,6 +8,11 @@ tasks:
     dir: '{{.USER_WORKING_DIR}}'
     cmds:
       - conan install . -s build_type=Debug -b missing -pr cpp20 -pr:b cpp20
+
+  lint:
+    dir: '{{.USER_WORKING_DIR}}'
+    cmds:
+      - cmake -D FORMAT_COMMAND=$(command -v clang-format || command -v clang-format-17 || command -v clang-format-12) FIX=YES -P cmake/lint.cmake
```
Comment (egelja) [inline linter/Taskfile.yml:15]:
```suggestion
      - cmake -D FIX=YES -P cmake/lint.cmake
```

update-alternatives exists for a reason

**Review by egelja — CHANGES_REQUESTED** [review]

File: linter/Taskfile.yml (hunk 1 above)
Comment (egelja) [reply linter/Taskfile.yml:15]: Also this should be task `format` if you do `FIX=YES`

Comment (andrlime) [reply linter/Taskfile.yml:15]: ok

**Review by egelja — APPROVED** [review]

**Comment by egelja** [comment]
squash pls

**Comment by andrlime** [comment]
> squash pls

Okay, once Steve approves cuz I see "merging is blocked"

**Pull request merged**